create/application.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/ninech/nctl/api/util"
 	"github.com/ninech/nctl/internal/format"
 	"github.com/ninech/nctl/logs"
+	k8sError "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
Nit: would name this `kerrors` for consistency (not yet in this repo but our mono-infra uses it like that)

Reply:
done